pkg/metadata/metadata.go

```diff
@@ -0,0 +1,4 @@
+package metadata
```

Comment:
I feel like there should be more in this package e.g. `PubKeyHashAnnotation`. If you agree, we can create a story to follow-up on this.